rpc/src/server/router.rs

```diff
@@ -255,6 +255,11 @@ pub(crate) fn create_routes(is_sandbox: bool, one_context: bool) -> PathTree<Met
         "/stats/memory",
         dev_handler::dev_stats_memory,
     );
+    routes.handle(
+        hash_set![Method::GET],
+        "/dev/stats/rw",
```

Comment:
@mambisi 
other stats starts  with `/stats/`
so I would change `/dev/stats/rw` to `/stats/storage/rw` or `/stats/maindb/rw`